src/interpreter-errors.ts

```diff
@@ -107,7 +107,10 @@ export class UndefinedVariable extends RuntimeSourceError {
   }
 
   public elaborate() {
-    return 'TODO'
+    return `Before you can read the value of ${
+      this.name
+    }, you need to declare it as a variable or a constant.
+		You can do this using the let or const keywords.`
```

Comment:
Is the second sentence supposed to be on a new line? If so you should get rid of the leading spaces. Spaces are preserved in backtick strings. (the ones surrounded by `)